package.json

```diff
@@ -21,13 +21,17 @@
     "supertest": "^6.0.1"
   },
   "dependencies": {
+    "axios": "^0.21.1",
+    "babel-eslint": "^10.1.0",
+    "config": "^3.3.3",
```

Comment:
Seems like `config` isn't needed anymore? But on the other hand, just to confirm - babel-eslint is still needed even with the upgrade to node 14?

Reply:
Removed config but babel-eslint is still required to make lint pass